lib/shopify-cli/tasks/authenticate_shopify.rb

```diff
@@ -5,30 +5,15 @@ module Tasks
     class AuthenticateShopify < ShopifyCli::Task
       def call(ctx)
         Tasks::EnsureEnv.call(ctx)
-        token = oauth_client.authenticate("https://#{env.shop}/admin/oauth")
-        @env = Helpers::AccessToken.write(token)
-        ctx.puts "{{success:Authentication Token written to env file}}"
-      rescue OAuth::Error => e
-        ctx.puts("{{error: #{e}}}")
-        ctx.puts("{{error:Failed to retrieve ID & Refresh tokens}}")
-        ctx.puts "{{*}} Remeber to add {{underline: #{oauth_client.redirect_uri} "\
```

Comment:
If we're getting rid of this are we surfacing this reminder anywhere else? Do we need to?

Reply:
I removed this because it was no longer working anyway. I might need to open an issue to track this. Since we put in the Spin group, these messages were swallowed. We might also need better top hatting guidelines so that this doesn't happen.

Reply:
Yikes yeah

Reply:
> better top hatting guidelines
can we catch stuff like this with tests?

also if we add it back we should fix the typo, `Remeber` --> `Remember`

Reply:
I am not sure, can we test Cli::UI output? like the spinners and stuff like that? It seems like it would be difficult. Maybe @tylerball or @katiedavis can answer this because I am not sure.

Reply:
I could bring these messages back into the authenticate command